pkg/api/norman/customization/cluster/validator.go

```diff
@@ -668,6 +669,15 @@ func validateGKENodePools(spec *v32.ClusterSpec) error {
 			errors = append(errors, fmt.Sprintf("node pool [%s] version cannot be empty", name))
 			continue
 		}
+
+		// Windows images are WINDOWS_LTSC or WINDOWS_SAC. The cluster must have at least one non-Windows node pool.
+		if !hasRequiredLinuxPool && !strings.Contains(strings.ToLower(np.Config.ImageType), "windows") {
```

Comment:
nit: checking hasRequiredLinuxPool isn't necessary.
```suggestion
		if !strings.Contains(strings.ToLower(np.Config.ImageType), "windows") {
```

Reply:
Well I don't disagree but this was @thedadams's suggestion https://github.com/rancher/rancher/pull/32171#discussion_r614437854